sites/all/themes/backdropapi/style.css

```diff
@@ -5,3 +5,18 @@ td.active {
 th {
   background-color: inherit !important;
 }
+
+h2 {
+  font-weight: 500;
+}
+code {
+  border: none;
+  background-color: #F7F7F7;
+  margin-bottom: 2em;
+}
+
+blockquote.d7diff {
```

Comment:
I think its best to keep it so we can style D7 differences differently from other differnces, if you understand what I mean. In time we could create a .bddiff callout block, give it a red background so its obvious etc.